apps/web/data/activities.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+        "tittle": "Mini sessão de abertura",
+        "details": "10:00 - 10:30 || Júlio Pinto - Organizador DojoCon",
+        "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer eros nisi, blandit id justo eget, auctor tincidunt est."
+    },
+    {
+        "tittle": "Apresentação CoderDojo Braga e Projetos",
```

Comment:
For the descriptions to the ones I said previously ask someone from the Communication team to write them for you :+1: